src/main/java/pageObjects/Cartpage.java

```diff
@@ -19,7 +19,7 @@ public Cartpage(WebDriver driver)
 By addtocheckoutbutton = By.name("proceedToRetailCheckout");
 By redirecttocartbutton = By.id("nav-cart-count-container");
 By searchbar = By.id("twotabsearchtextbox");
-public Checkoutpage addtocheckout()
+public Checkoutpage addtocheckout() 
```

Comment:
what is th purpose of this? I don't understand what's being done here

Reply:
so I mistakenly called the function addtocheckout twice,  so when I use that method, it clicks the button in the cart to go to checkout page, since I have defined a class for checkoutpage already, I thought of creating the object then and there, so that it returns the checkoutpage object and it can be directly used